shared/cmd/flags.go

```diff
@@ -250,3 +253,17 @@ func LoadFlagsFromConfig(cliCtx *cli.Context, flags []cli.Flag) error {
 	}
 	return nil
 }
+
+// ValidateNoArgs insures that the application is not run with erroneous arguments or flags.
```

Comment:
Cool side project idea: a static analyzer to enforce `ValidateNoArgs` before  `app.Before`